app/client/views/settings/users/enroll/quickformDropdown.js

```diff
@@ -0,0 +1,15 @@
+import SlimSelect from "slim-select";
```

Comment:
We don't need to duplicate this template, since we can re-use the component from the User Groups management form.

Reply:
Yes that was my intention, but it overrides the slim select css of edit groups if I pass the attributes from autoform.

Reply:
Where are the `selectOptions` generated?

Reply:
in the schema.js file

Reply:
Just use the Groups Widget from here:
https://github.com/GeriLife/wellbeing/tree/develop/app/client/views/settings/users/groups

Pass in the `selectOptions`.

Move the widget code to a "shared" directory.

Reply:
Since the schema is not attached to a collection, lets just use a template helper to prepare the data.

Reply:
Yes thats what I did but the SlimSelect css only applies on either one of the places  it is used.

Reply:
May be I can make 2 instances of slimselect. I am trying that

Reply:
The `allGroups` helper, mentioned above, can just return the groups:

```js
Template.usersEnroll.helpers({
  ...
  allGroups () {
    return Groups.find().fetch();
  },
  ...
```